src/main/java/org/zalando/failsafeactuator/endpoint/FailsafeEndpoint.java

```diff
@@ -23,15 +23,47 @@
 
   private static final String ENDPOINT_ID = "failsafe";
   private final CircuitBreakerRegistry circuitBreakerRegistry;
+  private final ApplicationContext context;
 
-  public FailsafeEndpoint(final CircuitBreakerRegistry circuitBreakerRegistry) {
+  public FailsafeEndpoint(final CircuitBreakerRegistry circuitBreakerRegistry, final ApplicationContext context) {
     super(ENDPOINT_ID);
     this.circuitBreakerRegistry = circuitBreakerRegistry;
+    this.context = context;
   }
 
   @Override
   public Map<String, CircuitBreakerState> invoke() {
     final Map<String, CircuitBreaker> breakerMap = circuitBreakerRegistry.getConcurrentBreakerMap();
+
+    if(!breakerMap.isEmpty()) {
```

Comment:
Why not both?

Reply:
Actually does not matter that much, as the old method will be deprecated anyway..

Reply:
Because I want to get rid of the map completely in long term.